mantidproject/mantid
# New splash screens and deprecation of Mantidplot

**Description of the work**
See commit messages, particularly the second one.


**To test:**
1. Start Mantiplot, look at the mantiplotplot specific "end of life" splash screen
1. Once started there should be a warning level message in the log about Workbench replacing mantidplot
1. The title bar should read "Mantidplot (End of life, use Workbench)", This should survive saving a project, loading a project and starting a new project.
1. Look over the changes to the release notes.
1. Start Workbench, see the workbench specific splash screen and no end of life message, also no end of life message in the log on startup.


Fixes #28719



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
